File: praktikum_algopascal/C/fungsi.c
#include <stdio.h>


int jml(int a, int b)
{
	return a + b;
}

int main()
{
	int x, y;
	printf("Masukan nilai X : ");
	scanf("%d", &x);
	printf("Masukan nilai Y : ");
	scanf("%d", &y);

	printf("Hasil : %d", jml(x, y));
	return 0;
}

File: praktikum_algopascal/pascal/fungsi.pas
program fungsi;
	uses crt;
		function tambah(a, b: real): real;	
			var
				s: real;

begin
	clrscr;
	s := a + b;
	tambah := s;
end;
		var
			a, b: Real;
begin
	clrscr;
	write('Masukan angka a: ');
	readln(a);
	write('Masukan angka b: ');
	readln(b);
	writeln('Hasil : ', tambah(a, b):10:0);
	readln;
end.